scoring/score.py

```diff
@@ -1,20 +1,64 @@
+# -*- coding: utf-8 -*-
+
+import itertools
 import collections
 
 
 class InvalidScoresheetException(Exception):
     pass
 
 
+POINTS_HELD = 1
+POINTS_GROUND = 2
+POINTS_PLATFORM = 5
+
+NUM_TOKENS = 16
+
+
 class Scorer(object):
     def __init__(self, teams_data, arena_data):
         self._teams_data = teams_data
         self._arena_data = arena_data
```

Comment:
The shape of `arena_data` is pretty opaque. I realise it's just loaded in JSON/YAML but it would be ideal to put this into some kind of class structure to make that obvious, typable, and remove some boilerplate lower. We could provide defaults rather than using `.get(..., default)` in a bunch of places.

Reply:
6daa9b3aec5ff32b843164a3d0bc9a42f77ab3d6 moves the defaults, I think that helps?

Reply:
It does help, although `arena_data` and `teams_data` still aren't super obvious in their function and structure.

Reply:
Yeah, I do tend to end up referring to https://github.com/PeterJCLaw/libproton and then https://github.com/PeterJCLaw/proton to remind myself of them.